rules/team-works-in-vertical-slices/rule.md

```diff
@@ -0,0 +1,30 @@
+---
+type: rule
+title: Does your team work in vertical slices?
+uri: team-works-in-vertical-slices
+authors:
+  - title: Adam Cogan
+  - title: Jack Pettit
+related:
+  - the-team-do-you-encourage-multi-skilled-teams-by-leaving-your-comfort-zone
+created: 2023-02-22T04:02:12.393Z
+guid: d85ead8c-9a6b-4715-b798-e3db535604f8
+---
+As a Software Developer you often work with both the front and backend. Most people like to work with which ever is their strong suit. 
+
+That doesn't help get your weaker skills stronger. Lets look at another way to increase your weaker skills.
+
+<!--endintro-->
+
+When working in teams it is often normal to split the work into what peoples strong suits are. This can be beneficial but it comes with a flaw of not helping developers strengthen their weaker skills. At SSW we prefer to do a whole slice of work each time (from the front end to the API, and any extras in between). 
+
+Part of Scrum means all developers are responsible for the work that is conducted during a sprint. To push this point, as part of our definition of done; Adding 2 other developers to look over the code allows for,
+
+1. The quality of code to remain high
+2. Still allows for all developers to learn (Even those reviewing) 
+3. Teaching the author of the PR 
+4. Asking questions
+
+
+![Figure: Vertical slicing is just like a slice of cake!](vertical-slicing-new-post-img-min.webp)
```

Comment:
Stick to png or jpg. Gatsby does fancy image perf optimisation stuff and prefers png or jpg.